test/kb_Metrics_server_test.py

```diff
@@ -180,300 +795,562 @@ def test_MetricsMongoDBController_process_parameters(self):
-        print(pformat(ret[0]['metrics_result'][0:10]))
+    # @unittest.skip("skipped test_MetricsMongoDBController_update_user_info")
+    def test_MetricsMongoDBController_update_user_info(self):
+        user_list = ['sulbha', 'ytm123', 'xiaoli', 'andrew78', 'qzhang']
+        start_datetime = datetime.datetime.strptime('2018-01-01T00:00:00+0000',
+                                                    '%Y-%m-%dT%H:%M:%S+0000')
+        end_datetime = datetime.datetime.strptime('2018-03-31T00:00:10.000Z',
+                                                  '%Y-%m-%dT%H:%M:%S.%fZ')
+
+        # testing update_user_info with given user_ids
+        params = {'user_ids': user_list}
+        params['epoch_range'] = (start_datetime, end_datetime)
+        upd_ret = self.db_controller._update_user_info(
+                self.getContext()['user_id'],
+                params, self.getContext()['token'])
+        self.assertEqual(upd_ret, (4, 0))
+
+        # testing update_user_info without user_ids
+        params = {'user_ids': []}
+        params['epoch_range'] = (start_datetime, end_datetime)
+        upd_ret = self.db_controller._update_user_info(
+                self.getContext()['user_id'],
+                params, self.getContext()['token'])
+        self.assertEqual(upd_ret, (33, 4))
 
     # Uncomment to skip this test
-    @unittest.skip("skipped test_run_get_user_ujs_results")
-    def test_run_get_user_ujs_results(self):
-        m_params = {
-            'user_ids': ['qzhang'],  # 'user_ids': [],
-            #'epoch_range':(1420083768000,1435677602000)#(datetime.datetime(2015, 1, 1), datetime.datetime(2015,6,30)
-            #'epoch_range':(1420083768000,1451606549000)#(datetime.datetime(2015, 1, 1), datetime.datetime(2016,1,1)
-            # (datetime.datetime(2015, 1, 1), datetime.datetime(2017,9,20)
-            'epoch_range': (1420083768000, 1505876263000)
-        }
-        # Second, call your implementation
-        ret = self.getImpl().get_user_ujs_results(self.getContext(), m_params)
-        print("Number of records get_user_ujs_results returned=" +
-              str(len(ret[0]['metrics_result'])))
-        print(pformat(ret[0]['metrics_result'][0:10]))
+    # @unittest.skip("skipped _update_daily_activities")
+    def test_MetricsMongoDBController_update_daily_activities(self):
+        start_datetime = datetime.datetime.strptime('2018-01-01T00:00:00+0000',
+                                                    '%Y-%m-%dT%H:%M:%S+0000')
+        end_datetime = datetime.datetime.strptime('2018-03-31T00:00:10.000Z',
+                                                  '%Y-%m-%dT%H:%M:%S.%fZ')
+
+        # testing update_daily_activities with given user_ids
+        params = {'epoch_range': (start_datetime, end_datetime)}
+        upd_ret = self.db_controller._update_daily_activities(
+                self.getContext()['user_id'],
+                params, self.getContext()['token'])
+        self.assertEqual(upd_ret, (1, 7))
```

Comment:
Please check those 7 records actually changes in DB. Also check prior to this call, there should be only 7 records satisfying update filter.